.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+    - "2.7"
+sudo: false
+
+cache:
+  - pip: true
+  - custom_install: true  # See https://github.com/travis-ci/travis-ci/issues/3239.
+before_install:
+    - "export DISPLAY=:99.0"
+    - "sh -e /etc/init.d/xvfb start"
+install:
+    - pip install -U pip wheel codecov
+    - pip install -r requirements/test.txt
+script:
+    - make validate
+branches:
```

Comment:
I have yet to find good documentation regarding why this is necessary. I personally like the idea of testing the branch as well as the branch merged into master, which is what happens when you remove this directive.